firmware/application/apps/ui_text_editor.cpp

```diff
@@ -413,16 +413,9 @@ TextEditorView::TextEditorView(NavigationView& nav)
     };
 
     menu.on_open() = [this]() {
-        /*show_save_prompt([this]() {
```

Comment:
This still doesn't work correctly due to focus getting called without parent_ set.